go.sum

```diff
@@ -27,6 +27,7 @@ contrib.go.opencensus.io/exporter/stackdriver v0.13.1/go.mod h1:z2tyTZtPmQ2HvWH4
 contrib.go.opencensus.io/resource v0.1.2 h1:b4WFJV8u7/NzPWHeTqj3Ec2AW8OGhtJxC/hbphIOvbU=
 contrib.go.opencensus.io/resource v0.1.2/go.mod h1:F361eGI91LCmW1I/Saf+rX0+OFcigGlFvXwEGEnkRLA=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
+git.apache.org/thrift.git v0.12.0/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg=
```

Comment:
Is this some fork of Thrift? Looks like it appeared after removing
```
replace git.apache.org/thrift.git v0.12.0 => github.com/apache/thrift v0.12.0
```
from go.mod above.

`go mod graph` shows that OpenCensus depends on it:
```
go.opencensus.io@v0.19.1 git.apache.org/thrift.git@v0.12.0
```
which is coming from this exporter:
```
github.com/signalfx/opencensus-go-exporter-kinesis@v0.4.2 go.opencensus.io@v0.19.1
```

Reply:
It is moved from line 96. Don't ask me why `go mod tidy` decided on this change.

Reply:
The latest Kinesis exporter release https://github.com/signalfx/opencensus-go-exporter-kinesis/releases/tag/v0.6.3 has a newer dependency on [OpenCensus v0.22.2](https://github.com/signalfx/opencensus-go-exporter-kinesis/blob/873a7ae3fb6d37cc78b8b692e4f1a8ceb89218bb/go.mod#L12) so updating it should eliminate this dependency.

Reply:
Sorry, that replace was given me an error after I deleted the scribe. That's why I removed it.

Reply:
@nilebox please send a PR after this.